src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -233,11 +190,74 @@ ApiGlobe.prototype.addElevationLayersFromJSON = function addElevationLayersFromJ
 ApiGlobe.prototype.addElevationLayersFromJSONArray = function addElevationLayersFromJSONArray(urls) {
     var proms = [];
 
-    for (var i = 0; i < urls.length; i++) {
-        proms.push(Fetcher.json(urls[i]).then(this.addElevationLayer.bind(this)));
+    for (const url of urls) {
+        proms.push(Fetcher.json(url).then(layer => this.addElevationLayer(layer)));
+    }
+
+    return Promise.all(proms);
+};
+
+function updateLayersOrdering(globeLayer, imageryLayers) {
```

Comment:
Currently this methods belongs to `ApiGlobe` so it expects a globeLayer.
I'll change the name anyway.